pkg/scheduler/framework/runtime/framework.go

```diff
@@ -1340,6 +1356,18 @@ func (f *frameworkImpl) ProfileName() string {
 	return f.profileName
 }
 
+// PercentageOfNodesToScore returns percentageOfNodesToScore associated to this framework.
+// If the profile level percentage is nil, return the scheduler (global) percentage.
+func (f *frameworkImpl) PercentageOfNodesToScore() int32 {
+	if f.percentageOfNodesToScore != nil {
+		return *f.percentageOfNodesToScore
+	}
+	if f.schedulerPercentageOfNodesToScore != nil {
```

Comment:
Theoretically, f.schedulerPercentageOfNodesToScore shouldn't be nil since we have defaulted its value. Is it because we use this in tests but we didn't run schema Defaults(), otherwise I think we should avoid this for this may be a bug in Defaults(), we shouldn't bury it.

Reply:
That's a very good point.  Removed `if f.schedulerPercentageOfNodesToScore != nil {`